kubernetes/browser.yaml

```diff
@@ -62,12 +69,18 @@ kind: Ingress
 metadata:
   name: browser
   namespace: buildbarn
+  annotations:
```

Comment:
My recommendation would be to remove these annotations and the TLS options as well, as they are highly dependent on how people provision clusters. For example, many people use Traefik nowadays.